src/main/resources/application.yml

```diff
@@ -32,3 +32,6 @@ redis :
 #                 category: "company"
 #                 version: 2
 
+
+fdk-extension:
+  version: 0.5.0-beta.1
```

Comment:
why we don't have 0.6.0-beta? @meetkoriya13

Reply:
@brijeshgajjarfynd we have not created the 0.6.0-beta version on Java library yet.